apps/web/screens/Verdicts/VerdictsList.tsx

```
@@ -148,19 +264,531 @@ const VerdictsList: CustomScreen<VerdictsListProps> = ({
+                    return {
+                      ...previousState,
+                      [QueryParam.CASE_CATEGORIES]:
+                        updatedCaseCategories.length === 0
+                          ? null
+                          : updatedCaseCategories,
+                    }
+                  },
+                )
+              }}
+            />
+          ))}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)